project-templates/ecs/basic/README.md

```diff
@@ -41,32 +41,26 @@ Prerequisites:
    <br>
    <br>
 
-### AWS Steps with Terraform
+### AWS Steps with Terraform CLI
 
 Create the baseline infrastructure with the following steps in your CLI.
 
 #### Terraform AWS Example Microservice Infra
-1. Navigate to the `terraform-aws-example-microservice-infra` directory found in this example.
-   <br>
-   <br>
-2. Update the `main.tf` file with the `{{YOUR_TERRAFORM_ORGANIZATION_NAME}}` field and replace the value with your own Terraform Cloud organization name.
-   <br>
-   <br>
-3. Run `terraform init` and the `terraform apply -var=vpc_name=example-cluster` command in this directory.
-   <br>
-   <br>
-   <br>
+1. Clone the [hashicorp/waypoint-examples](https://github.com/hashicorp/waypoint-examples) repositroy to your local machine and navigate to this example project under `/project-templates/ecs/basic` (you may need to first checkout the `project_templating_basic_example` branch)
+1. Navigate to the `terraform-aws-example-microservice-infra` directory found in this example.  
+  - `$ cd terraform-aws-example-microservice-infra`
+
+1. Update the `main.tf` file to replace the `{{YOUR_TERRAFORM_ORGANIZATION_NAME}}` placeholder with the value of your Terraform Cloud organization name and save the file.
+1. Run `terraform login` and follow the steps to generate and save a TFC credential file. 
+1. Run `terraform init` and then run `terraform apply -var=cluster_name=example-cluster` command in this directory.
+1. Terraform Cloud will now create the needed infrastructure based on your TFC Organization name. You can monitor the progress in Terraform Cloud by clicking on the `aws-example-microservice-infra` workspace.
```

Comment:
```suggestion
1. Clone the [hashicorp/waypoint-examples](https://github.com/hashicorp/waypoint-examples) repositroy to your local machine and navigate to this example project under `/project-templates/ecs/basic` (you may need to first checkout the `project_templating_basic_example` branch)
2. Navigate to the `terraform-aws-example-microservice-infra` directory found in this example.  
  - `$ cd terraform-aws-example-microservice-infra`

3. Update the `main.tf` file to replace the `{{YOUR_TERRAFORM_ORGANIZATION_NAME}}` placeholder with the value of your Terraform Cloud organization name and save the file.
4. Run `terraform login` and follow the steps to generate and save a TFC credential file. 
5. Run `terraform init` and then run `terraform apply -var=cluster_name=example-cluster` command in this directory.
6. Terraform Cloud will now create the needed infrastructure based on your TFC Organization name. You can monitor the progress in Terraform Cloud by clicking on the `aws-example-microservice-infra` workspace.
```

Reply:
So _normally_ markdown would convert the sequence of `1.` automatically, but in this case I broke it up with the `- $ cd` part so yeah numbering directly here is a good idea